library/src/interactor.cxx

```diff
@@ -31,6 +31,10 @@ const std::vector<std::pair<std::string, std::string> >& interactor::getDefaultI
     { "L", "Increase (+Shift: decrease) lights intensity" },
     { "H", "Toggle cheat sheet display" },
     { "?", "Dump camera state to the terminal" },
+    { "1", "Front View camera" },
+    { "3", "Left View camera" },
+    { "7", "Top View camera" },
+    { "9", "Isometric View camera" },
     { "Escape", "Quit" },
     { "Enter", "Reset camera to initial parameters" },
     { "Space", "Play animation if any" },
```

Comment:
reorg as in other location

Reply:
I didn't understand what location are you refering to. Since `library/src/interactor.cxx` and `library/VTKExtensions/Rendering/vtkF3DRenderer.cxx` already follow different orders from each other, I decided to group every the new `1, 3, 7, 9` hotkeys with the other character hotkeys in the `interactor.cxx` file. Then `Escape`, `Enter`, etc would follow.